setup.py

```diff
@@ -31,7 +31,7 @@
         "fcm_django/migrations",
     ],
     python_requires=">=3.7",
-    install_requires=["firebase-admin>=5,<7", "Django"],
+    install_requires=["firebase-admin>=5,<7", "Django", "swapper"],
```

Comment:
If the version in requirements is pinned, should it be pinned here too?

Reply:
We don't pin Django here and pin it in requirements. I though do the same with swapper.

Reply:
Yeah. It's strange that firebase is pinned and django is not